src/Sylius/Bundle/CoreBundle/Kernel/Kernel.php

```diff
@@ -131,6 +132,18 @@ public function registerBundles()
     /**
      * {@inheritdoc}
      */
+    protected function getContainerBaseClass()
+    {
+        if ('test' === $this->environment) {
+            return '\PSS\SymfonyMockerContainer\DependencyInjection\MockerContainer';
```

Comment:
It's already included in `app/AppKernel.php` and `app/TestKernel.php`

Btw. is it needed in this PR?